Library/Formula/libhttpserver.rb

```diff
@@ -0,0 +1,41 @@
+require "formula"
+
+class Libhttpserver < Formula
+  homepage "https://github.com/etr/libhttpserver"
+  url "https://github.com/etr/libhttpserver/archive/v0.7.1.tar.gz"
+  head "https://github.com/etr/libhttpserver.git"
+  sha1 "b36c8a5f523c04613d6a36c86bbb31a2b92aab83"
+
+  option "enable-cpp11", "Enable c++11 std classes"
+  depends_on "libmicrohttpd"
+
+  depends_on "pkg-config" => :build
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+
+
+  option :universal
+
+  def install
+    ENV.universal_binary if build.universal?
+
+    ENV["LIBMICROHTTPD_CFLAGS"] = '$(pkg-config libmicrohttpd --cflags)'
+    ENV["LIBMICROHTTPD_LIBS"] = '$(pkg-config libmicrohttpd --libs)'
```

Comment:
Also: this should use backticks instead of embedding a bash command inside a environment variable.